src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1059,7 +1059,7 @@ public function morphs($name, $indexName = null)
     {
         $this->unsignedInteger("{$name}_id");
 
-        $this->string("{$name}_type");
+        $this->string("{$name}_type", 191);
```

Comment:
Not sure it makes sense to hard-code this value here, especially since this issue is database engine-specific.

See: https://github.com/laravel/framework/issues/17508

Reply:
Thanks.
I could not find this workaround.